components/member-profile/index.js

```diff
@@ -1,18 +1,18 @@
-import { getDataURL, getImgURL } from 'helper-functions/urls';
-
-import PropTypes from 'prop-types';
 import React from 'react';
-import classNames from './member-profile.module.scss';
+import PropTypes from 'prop-types';
```

Comment:
Was this used?

Reply:
Yes it is used on the line number 21

Reply:
Oh. My bad. How was this used before importing earlier 🤦

Reply:
Not sure 😁